src/api/common_api.js

```diff
@@ -810,6 +812,44 @@ module.exports = {
                     $ref: '#/definitions/fs_backend'
                 }
             }
-        }
+        },
+        nsfs_account_config: {
+            type: 'object',
+            properties: {
+                uid: { type: 'string' },
+                gid: { type: 'string' },
```

Comment:
@jeniawhite @romayalon I think this should be `type: 'number'`.
Any reason this should be a string?
Even with number we can check wether it's undefined vs. 0 if we need to support uid/gid of root.

Reply:
@guymguym Agree, it should be a Number.
I do not see a reason for it to be a String.